mmv1/products/compute/Network.yaml

```diff
@@ -224,10 +224,13 @@ properties:
     default_from_api: true
   - !ruby/object:Api::Type::Enum
     name: 'networkFirewallPolicyEnforcementOrder'
+    update_verb: :PATCH
+    update_url: projects/{{project}}/global/networks/{{name}}
     description: |
-      Set the order that Firewall Rules and Firewall Policies are evaluated. Needs to be either 'AFTER_CLASSIC_FIREWALL' or 'BEFORE_CLASSIC_FIREWALL' Default 'AFTER_CLASSIC_FIREWALL'
-    immutable: true
+      Set the order that Firewall Rules and Firewall Policies are evaluated.
+      Needs to be either `AFTER_CLASSIC_FIREWALL` or `BEFORE_CLASSIC_FIREWALL`.
+      Defaults to `AFTER_CLASSIC_FIREWALL`.
     values:
       - :BEFORE_CLASSIC_FIREWALL
       - :AFTER_CLASSIC_FIREWALL
-    default_value: :AFTER_CLASSIC_FIREWALL
+    default_from_api: true
```

Comment:
Why are we changing from default value to optional computed? Does the default value here not match with the API default for the field?

Reply:
@shuyama1 sorry my bad. I noticed other values like `mtu` default from the API so I applied the same here. I understand that we want a computed value when it's a complex default (eg. API defaults to `X` if setting `A` is enabled otherwise defaults to `Y`) but then why is `mtu` defaulting from API when its a simple default of 1460?